README.md

```diff
@@ -288,8 +308,9 @@ flowchart LR
 
 ## Signals
 
-This app provides a single `wagtail_localize.signals.translation_imported`
-signal that is sent when translation are imported from Smartling.
+This app provides two Signals.
+
+`wagtail_localize.signals.individual_translation_imported` is sent when translation are imported from Smartling.
```

Comment:
The tense feels weird here. When _a_ translation is imported? When translations are imported?